virttest/env_process.py

```diff
@@ -1037,7 +1037,7 @@ def preprocess(test, params, env):
         brcfg.setup()
 
     base_dir = data_dir.get_data_dir()
-    if params.get("storage_type") == "iscsi":
+    if params.get("storage_type") == "iscsi" and vm_type != "libvirt":
```

Comment:
I think "lvm" type prepare can also be ignored for libvirt as I saw it also use qemu_storage function. Could you please deal with it also?

Reply:
Agree. We can also fix lvm part which is definitely not used by libvirt cases.